docs/README.md

```diff
@@ -0,0 +1,36 @@
+# Nebula docs
+---
+
+**Nebula** is a distributed, scalable, lightning-fast graph database. It is the only solution in the world capable to host graphs with dozens of billions of vertices (nodes) and trillions of edges, while still provides millisecond latency.
+
+**Nebula's** goal is to provide reading, writing, and computing with high concurrency, low latency for super large scale graphs. Nebula is an open source project and we are looking forward to working with the community to popularize and promote the graph database.
+
+As a graph database, **Nebula** has these features
+* Symmetrically distributed
+* Highly scalable
+* Fault tolerant
+* Strong data consistency
+* SQL-like query language
+
+## How can I get Nebula ##
+**Nebula** source code is available here on [`GitHub`](https://github.com/vesoft-inc/nebula). The currently release is version 0.1. Please refer the `Release Notes` here for details. You can also download [`docker` ](https://www.docker.com/get-started)image to try it.
```

Comment:
Add some line breaks？

Reply:
Thanks for your suggestion, but since the two sentences are associated, it's better to put them together.

Reply:
_**Nebula** source code is available here on [`GitHub`](https://github.com/vesoft-inc/nebula). The currently release is version 0.1. Please refer the `Release Notes` here for details_ ;  "**Nebula** source code" is a little bit weird

Reply:
> _Fault tolerant_ or _Fault-tolerant_?

I researched some pages on the use of the hyphen, an adj. + n. should use a hyphen, for example, the 'fault-tolerant' here, but when in polysyllabic words, a hyphen is not suggested. That's why I didn't use it here. What's more, if you are not sure when to use a hyphen, just don't use it because the use of hyphens is becoming less and less.

Reply:
> _**Nebula** source code is available here on [`GitHub`](https://github.com/vesoft-inc/nebula). The current release is version 0.1. Please refer the `Release Notes` here for details_ ; "**Nebula** source code" is a little bit weird

Unfortunately, we don't have release notes yet, what's more, this version might not be reserved in the future, thank you.